outreach/gsoc/2023/gpu-acceleration-in-trixi-jl-using-cuda-jl.md

```diff
@@ -0,0 +1,113 @@
+@def title = "GSoC 2023: GPU acceleration in Trixi.jl using CUDA.jl"
+@def maxtoclevel=2
+
+# GSoC 2023: GPU acceleration in Trixi.jl using CUDA.jl
+
+ 
+- Mentee: [Huiyu Xie](https://github.com/huiyuxie)
+- Mentors: [Hendrik Ranocha](https://github.com/ranocha) and [Michael Schlottke-Lakemper](https://github.com/sloede)
+- Project Link: https://github.com/huiyuxie/trixi_cuda
+
+
```

Comment:
```suggestion
The goal of this GSoC project was to accelerate Trixi.jl using GPUs.

**Table of contents**
\toc

```
I suggest to add a one-line summary (feel free to improve it) and a table of contents for more convenient reading. This will look something like the following:
<img width="722" alt="image" src="https://github.com/trixi-framework/trixi-framework.github.io/assets/3637659/d578f03b-3ce2-494a-924b-0f6aa7356bf0">